client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useCallback, useRef } from 'react';
+import { __ as NO__ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import Suggestions from 'components/suggestions';
```

Comment:
Why not directly use the `SuggestionSearch` search component? :slightly_smiling_face: 

> SuggestionSearch is a bundled component of FormTextInput and Suggestions which encapsulates the common boilerplate code for pairing them.

Reply:
That was actually the first attempt but it didn't provided enough flexibility to match our design guidelines (eg: showing popular suggestions when the query is empty). Of course we can try to do some changes there to support our needs but there is also the problem of extracting it to `@automattic/components`. `Suggestions` comes with just analytics as a dependency while `SuggestionSearch` is importing 4 more components that are not currently `@automattic/components`.